platform/sun50i/devices.c

```diff
@@ -90,3 +92,11 @@ static struct device r_timer0 = {
 	.irq      = IRQ_R_TIMER0,
 	.irqdev   = &r_intc,
 };
+
+static struct device r_twd = {
+	.name     = "r_twd",
+	.clock    = CCU_GATE(R_CCU_GATE_R_TWD),
```

Comment:
This needs to be changed now that #63 is merged (`R_CCU_CLOCK_R_TWD`)

Reply:
Done